src/core/Akka.Cluster.Tests/MemberOrderingSpec.cs

```diff
@@ -94,6 +94,42 @@ public void MemberOrdering_must_consistent_ordering_and_equals()
         }
 
         [Fact]
+        public void MemberOrdering_must_work_pick_member_with_legal_transition()
```

Comment:
Smaller sanity check spec to verify that two collections of `Member`s with both different addresses and statuses for the same address can be merged correctly.